mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -2592,6 +2593,8 @@ func TestClientSideEncryptionProse(t *testing.T) {
 
 		for _, test := range tests {
 			mt.Run(test.typeStr, func(mt *mtest.T) {
+				mt.Skipf("Skipping test until GODRIVER-3123 is implemented")
+
```

Comment:
```suggestion
```

Can we remove this if we make the MaxServerVersion 7.99.99? I missed this requirement when implementing DRIVERS-2861.